app/src/main/java/com/nutomic/syncthingandroid/service/SyncthingService.java

```diff
@@ -587,6 +587,13 @@ public NotificationHandler getNotificationHandler() {
         return mNotificationHandler;
     }
 
+    public String getRunDecisionExplanation() {
+        if (mRunConditionMonitor == null) {
+            return "This should not happen: mRunConditionMonitor is not instantiated.";
```

Comment:
Also don't hardcode messages, this should be in strings.xml.

Reply:
This is an exception because with the current PR's code,  this line is ensured to be never reached. It only fulfills the purpose of helping a later developer touching the code to notify him of a bad mistake.